src/pages/en/tutorial/2-pages/1.mdx

```diff
@@ -70,7 +70,7 @@ To make it easier to preview all your pages, add HTML page navigation links befo
 Check that you can click these links to move back and forth between pages on your site.
 
 :::note
-Astro uses standard HTML `<a>` elements to navigate between routes, with traditional page refreshes.
+Unlike many frameworks, Astro uses standard HTML `<a>` elements to navigate between pages, with traditional page refreshes.
```

Comment:
```suggestion
Unlike many frameworks, Astro uses standard HTML `<a>` elements to navigate between pages (also called _routes_), with traditional page refreshes.
```
@sarah11918 How about this, so it feels like a direct (rather than by-the-way) introduction?

Reply:
Ship it!